Add guidance on writing headings and using heading styles (#52)
springernature/ux-elements-docs: nature/styleguide/typography.md

* Body text and paragraphs at base sizes should not exceed 800px in width
* We use fluid typography by applying properties for different screen sizes, "Max" and "Min" 

## Headings

## Write headings in sentence case

For example, “Latest reviews and analysis” not “Latest Reviews and Analysis”.

Sentence case is easier for people to read.

Headings should follow [our content guidelines](/content).

### Make headings meaningful

Use descriptive headings that convey the meaning of the section they refer to. This helps users scan and find the information they need, without having to read all of the content from top to bottom.
For example, “Publish with us” is clearer than “Information on submission”.

### Use heading levels to communicate page structure

> Use semantic heading levels <h1> to <h6> to show the order of importance of the content on a page.

Reviewer: I try to avoid the term "importance" regarding heading levels. Something might be very important but detined to be (only) an `<h4>` just because of the structure. Perhaps:

_Use semantic heading levels `<h1>` to `<h6>` to indicate the relationship of sections to subsections in the page structure._

Also, use backticks for encapsulating code (\`&lt;h1>\`). I think those tags will be parsed as actual HTML if you don't mark them up as code snippets.

Author: Good call. How's this...

```suggestion
Use semantic heading levels `<h1>` to `<h6>` to show the relationship of sections to subsections in the page structure.
```

Reviewer: 👍

> Only include one <h1> per page, for the main heading.

Reviewer: ```suggestion
Only include one `<h1>` per page, for the main heading.
```

> Don’t skip heading levels. For example, don’t follow an <h2> with an <h4>. Doing this can be confusing, particularly for some assistive technology users.

Reviewer: ```suggestion
Don’t skip heading levels. For example, don’t follow an `<h2>` with an `<h4>`. Doing this can be confusing, particularly for some assistive technology users.
```

        </tr>	
</table>	

### Changing default heading sizes

> Our typography styles determine the size of headings based on the heading level you apply. For example, an <h1> is the largest at 30px, followed by <h2> at 24px, and so on.

Reviewer: ```suggestion
Our typography styles determine the size of headings based on the heading level you apply. For example, an `<h1>` is the largest at 30px, followed by `<h2>` at `24px`, and so on.
```

> If you need to use a different font size for a heading, you can use a utility class to override the default sizing.

Reviewer: Could have a code example here?

```
<h2 class="u-h3">Smaller h2 rendering</h2>
```

> If you need to use a different font size for a heading, you can use a utility class to override the default sizing.

Author: Does this work?

```suggestion
If you need to use a different font size for a heading, you can use a utility class to override the default sizing. For example:

`<h2 class="u-h3">Smaller h2 rendering</h2>`
```

Reviewer: Will need to be a code block, but yes! Like this:

```
```
<h2 class="u-h3">Smaller h2 rendering</h2>
```
```

Reviewer: (excuse the backslashes, I'm trying to escape it and failing)

Author: OK cool - I'll update. Thanks!
